joomla/joomla-cms
# [5.0] Fixes the cli installer

Pull Request for Issue #42114 .

### Summary of Changes

- refactor the code so cli installation doesn't throw 

### Testing Instructions

Install joomla using `cd installation` and `php joomla.php install`.
Make sure that you set the public folder option `--public-folder` to a valid directory (or do not provide any options and follow the questions but provide a valid path for the public folder)

### Actual result BEFORE applying this Pull Request

DB error

### Expected result AFTER applying this Pull Request

No errors

### Link to documentations
Please select:
